data/events.txt

```diff
@@ -1382,6 +1382,14 @@ event "initial deployment 4"
 		fleet "Large Southern Pirates" 2000
 		fleet "Small Free Worlds" 3000
 		fleet "Large Free Worlds" 5000
+	system "Shaula"
+		fleet "Small Southern Pirates" 500
+		fleet "Large Southern Pirates" 800
+		fleet "Small Core Pirates" 4000
+		fleet "Large Core Pirates" 7000
+		fleet "Small Northern Pirates" 5000
+		fleet "Large Northern Pirates" 9000
+		fleet "Large Militia" 9000
```

Comment:
This is identical to the default fleets for Shaula, and this is the first event that would change those fleets, so I don't think it's necessary.